pandas/conftest.py

```diff
@@ -1367,6 +1366,25 @@ def any_string_dtype(request):
     return request.param
 
 
+@pytest.fixture(
+    params=[
+        "object",
+        "string[python]",
+        pytest.param("string[pyarrow]", marks=td.skip_if_no("pyarrow")),
+        pytest.param("string[pyarrow_numpy]", marks=td.skip_if_no("pyarrow")),
+        pytest.param(pd.ArrowDtype(pa.string()), marks=td.skip_if_no("pyarrow")),
```

Comment:
I would actually prefer to just add the `pd.ArrowDtype(pa.string())` to the existing string dtypes instead of copying and creating a new fixture. Guessing that causes a lot of other test failures?

Reply:
Ah, I forgot that `pd.ArrowDtype(pa.string())` was not actually in the fixture, so my suggestion lead you a bit in the wrong way. Sorry!

Right now adding this to the main `any_string_dtype` fixture will indeed give quite some failures, I think. I agree that it might be better to actually do that (and it would be interesting to see which tests actually fail), but that's for another PR / out of scope for this bug fix (doing so would also require removing some tests are now only exist for the arrow string dtype, to not keep things duplicated).

Reply:
Yes, a lot of other tests need to be adjusted if adding ArrowDtype to the fixture.

So for this PR, should I just add test in `pandas/tests/extension/test_arrow.py`?

Reply:
Opened https://github.com/pandas-dev/pandas/issues/58495 so we can track the larger issue